### PR TITLE
Assorted doc bug fixes

### PR DESCRIPTION
**Pull request by jacobhammons**
Refs #4381
Refs #8146
Refs #13036
Refs #6853
Refs #9319
Refs #13777
Refs #2229
Refs #14946
Refs #15042

### CODE REVIEW

File: doc/ref/configuration/master.rst [hunk 1]
```diff
@@ -2668,6 +2668,32 @@ This can be used to control logging levels more specifically. See also
...
+
+A sample log event is as follows:
+
+.. code-block:: yaml
+
+    [DEBUG   ] ZeroMQ event: {'endpoint': 'tcp://127.0.0.1:4505', 'event': 512,
+    'value': 27, 'description': 'EVENT_DISCONNECTED'}
+
+All events logged will include the string ``ZeroMQ event``. A connection event
+should be logged on the as the minion starts up and initially connects to the
```
Comment (rallytime) [inline doc/ref/configuration/master.rst]: Missing a word here `logged on the as`

File: doc/ref/configuration/master.rst [hunk 2]
```diff
@@ -2668,6 +2668,32 @@ This can be used to control logging levels more specifically. See also
...
 
+.. conf_master:: zmq_monitor
+
+``zmq_monitor``
+---------------
+
+Default: ``False``
+
+To diagnose issues with minions disconnecting or missing returns, ZeroMQ
+supports the use of monitor sockets # to log connection events. This
```
Comment (rallytime) [inline doc/ref/configuration/master.rst]: Was the `#` intended here?

**Comment by jacobhammons** [comment]
@rallytime I fixed both the typos in the explanation, I also realized that I put this explanation in the master config, not the minion, so I corrected that too. Thanks for reviewing.

**Comment by rallytime** [comment]
@jacobhammons Oh, good catch. I didn't notice that. Looks good!

**Pull request merged**